src/main.py

```diff
@@ -178,6 +185,60 @@ def run_plan_bot(
             skip_tracking=skip_tracking,
         )
 
+    params = {}
+    if comments_progress:
+        newest_comment_id = comments_progress.get("newest")
+        oldest_comment_id = comments_progress.get("oldest")
+
+        if newest_comment_id:
+            params["before"] = latest_comment_id
+
+    # Get comments newer than latest
+    current_newest_comment_id = None
+    for comment in subreddit.comments(params=params):
+        comment = reddit_util.Comment(comment)
+        process_post(
+            comment,
```

Comment:
One big difference between this method and the other search methods is that it doesn't check for "WarrenPlanBot" at all. Based on the logic in `process_post`, this means that _every_ comment to /r/ElizabethWarren would be added to our db which is probably overkill, and would also mean that loading all the `post_ids_processed` with the check for `post.id in post_ids_processed` might become slow over time.

I think it might make sense to do some simple filtering on the comments here, so that if `warrenplanbot` isn't anywhere in the comment, we don't process it at all. We don't need to worry about wasting time reprocessing it either, since we'll be updating the cursor. (In contrast to the other methods which are cursorless)

Reply:
Makes sense. I _was_ thinking about the growth of `post_ids_processed` and wondering if it'd be performant enough to just check the `posts_db` instead. But I'm not gonna worry about it until you do 🤷‍♂. There probably aren't a ton of stuff in there just yet.